packages/templates/app/typescript-react/package.json

```diff
@@ -14,8 +14,8 @@
     "@polywrap/react": "0.10.0-pre.6",
     "http-proxy": "1.18.1",
     "http-proxy-middleware": "1.0.6",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
+    "react": ">=18.0.0",
```

Comment:
I'd prefer we revert this. If you merge the latest `origin-dev` branch, there is a patch fix for the `yarn` install error that these packages were causing: https://github.com/polywrap/toolchain/blob/6d2355f25c78597e4a125bf9818dd31daf0b2de9/package.json#L74-L77

Reply:
Oh great! Thanks for letting me know! I just patched it like that because `yarn install` was failing, and at that time the fix wasn't there. Since this worked for me, I didn't want to dig deep given that this is a draft anyway